tokio-trace/src/span.rs

```diff
@@ -1,6 +1,50 @@
-//! Spans represent periods of time in the execution of a program.
+//! Spans represent periods of time in which a program was executing in a
+//! particular context.
```

Comment:
I thought a span explicitly did _not_ establish context, and was _only_ a period of time? Anything else is up to the subscriber, no?